pkg/sbom/spdx/marshal.go

```diff
@@ -157,6 +158,21 @@ func (m *Marshaler) Marshal(r types.Report) (*spdx.Document2_2, error) {
 		}
 	}
 
+	if len(relationShips) > 1 {
+		// for consistent report generation accross UI and CLI
+		// sort relationships except for the first item
```

Comment:
Please add some more context in the comment i.e why first item is skipped and may be a small sample for better understanding

Reply:
Addressed